tests/Oryx.Integration.Tests/DevStorageAccountSanityTest.cs

```diff
@@ -23,7 +23,7 @@ public class DevStorageAccountSanityTest : StorageAccountSanityTestBase
                 output, 
```

Comment:
I can't remember... are we using the dev storage account at all after all of this? If so, I would keep this test and create a new one called StagingStorageAccountSanityTest. If not, then maybe we should just rename this?  

I also would like to point out that we still have the default set to `SdkStorageConstants.DevSdkStorageBaseUrl` in this constructor.